src/foam/u2/dialog/SimpleActionDialog.js

```diff
@@ -0,0 +1,79 @@
+foam.CLASS({
+  package: 'foam.u2.dialog',
+  name: 'SimpleActionDialog',
+  extends: 'foam.u2.Controller',
+
```

Comment:
This needs a foam header and documentation for this as well.

Reply:
Added